pkg/features/kube_features.go

```diff
@@ -1275,6 +1296,11 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 		{Version: version.MustParse("1.27"), Default: false, PreRelease: featuregate.Alpha},
 	},
 
+	KubeletPodResourcesListUseActivePods: {
+		{Version: version.MustParse("1.0"), Default: true, PreRelease: featuregate.GA},
```

Comment:
Should this be false, then marked as default true in 1.34?

Reply:
thanks @rphillips , the approach you suggested aligns more with how the deprecated FGs, even the one guarding fixes, seems to be used. Changing as suggested.